### ISSUE TITLE
Main menu. Remove duplication of main menu

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1841]
## Description of issue

Describe what is failing
- our main navigation is not responsive. We use two distinct nav elements to generate the same nav twice, showing up either on mobile or desktop breakpoints. 
- 

## Potential fix

- Ideally, we use one nav to avoid duplication of links
- We style the menu to work in both mobile and desktop breakpoints. 

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria

- [ ] Remove templates specific to mobile and desktop navigation
- [ ] Replace with a single responsive nav template
- [ ] Look into DS patterns to replicate, so we keep our header component aligned with mandatory elements. (Signature, language toggle and sign-in button)

**Comment by yaelberger-commits** [comment]
@amazingphilippe to pick up when you're back

**Comment by amazingphilippe** [comment]
Had to revert the previous PR. New PR fixing the issue https://github.com/cds-snc/notification-admin/pull/2274

**Comment by schancase** [comment]
fyi @connorscarolyns  - during Thursday standup with @yaelberger-commits  and @amazingphilippe , we decided to move this into the Blocked pipeline while Phil is away; until Aug.26

**Comment by yaelberger-commits** [comment]
Moving this back into the Sprint Backlog for @amazingphilippe to pick up again today

**Comment by amazingphilippe** [comment]
PR updated, pinged devs in Slack

**Comment by schancase** [comment]
has now been reviewed, just working on some outstanding issues/fixes

**Comment by amazingphilippe** [comment]
Merged!

**Comment by amazingphilippe** [comment]
Reoppening to work on the reverted fixes.

**Comment by schancase** [comment]
Jumana proposing this gets merged by a dev on a Monday

**Comment by schancase** [comment]
Andrew to review and merge PR when able, on a Monday

**Comment by schancase** [comment]
plan is to merge this week of Feb.23

**Comment by schancase** [comment]
moving this back to product backlog. We will pull as part of next sprint at above-planned date.

**Comment by schancase** [comment]
let's plan to merge on Monday Feb.23 (Jumana)

**Comment by schancase** [comment]
planning to merge this today and release tomorrow

**Comment by schancase** [comment]
update-will release this tomorrow
